tools/docker.go

```diff
@@ -94,6 +82,14 @@ func DockerRun(ctx context.Context, args []string, volumes []string) error {
 		if abs, err := filepath.Abs(v); err != nil {
 			return fmt.Errorf("could not find volume path '%s': %v", v, err)
 		} else {
+			// fix windows paths to contain only slashes
```

Comment:
Same as above. Can you extract a method for this to avoid repetition?

Reply:
Done